docs/fiddles/features/notifications/renderer/renderer.js

```diff
@@ -1,7 +1,6 @@
-const myNotification = new Notification('Title', {
-  body: 'Notification from the Renderer process'
-})
+const NOTIFICATION_TITLE = 'Title'
+const NOTIFICATION_BODY = 'Notification from the Renderer process. Click to log to console.'
+const CLICK_MESSAGE = 'Notification clicked'
 
-myNotification.onclick = () => {
-  console.log('Notification clicked')
-}
+new Notification(NOTIFICATION_TITLE, { body: NOTIFICATION_BODY })
+  .onclick = () => console.log(CLICK_MESSAGE)
```

Comment:
IMO, `console.log(...)` isn't super obvious (especially since we have two) - would it be better to just update some text in the UI?

Reply:
Thanks you! I felt the same way, but am glad to have the confirmation. When working with the _main_ notification feature (PR 29268), I noticed that updating the UI would add a bit of complexity because you have to use an IPC to communicate with the UI. I'll ask in that PR, but I'm wondering if we should add that complexity in order to update the UI or stick with the console on that one.

Reply:
I don't think that one takes any action/response when the notification is clicked (maybe this is a miss?) - I think adding IPC from main back to renderer there probably over-complicates the example?

Reply:
Done in `bccc`